src/components/Input/index.js

```diff
@@ -153,7 +153,11 @@ class Input extends React.Component {
   }
 
   isAutoFocus() {
-    return this.props.autoFocus && !this.props.disabled && !this.props.readonly;
+    return (this.props.autoFocus || this.props.forceFocus) && !this.props.disabled && !this.props.readonly;
+  }
+
+  isForceFocus() {
```

Comment:
one small note, maybe we should call it `should(Force/Auto)Focus`?

Reply:
As this is an internal function, let's do it in separate PR.